src/data-access/FetchSearchEntityRepository.ts

```diff
@@ -39,8 +40,12 @@ export default class FetchSearchEntityRepository implements SearchEntityReposito
 			url.searchParams.set( key, params[ key ] );
 		}
 		let response: Response;
+		const debouncedFetch = debounce( fetch, 300, {
+			leading: true,
+			trailing: false,
```

Comment:
No, I think we want the trailing edge. I.e. if I type four keys after each other, then I want to see the suggestions for the search result with four letters instead of one

Reply:
Done